### PR TITLE
To do list with React

### PR DESCRIPTION
**Pull request by fed1k**
* Add new list to the lists
* Remove list from lists
* Edit list from lists
* Hamburger button fucntionality
* Save data in the local storage

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,70 @@
...
+
+Launches the test runner in the interactive watch mode.\
+See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+
+### `npm run build`
+
+Builds the app for production to the `build` folder.\
+It correctly bundles React in production mode and optimizes the build for the best performance.
+
+The build is minified and the filenames include the hashes.\
```
Comment (ocrearyy) [inline README.md]: Maybe consider updating the readme so it looks more professional.

File: src/App.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import './index.css';
+import React from 'react';
+import Todo from './components/Todo';
+
+function App() {
+  return (
+    <div className="App">
+      <Todo />
+    </div>
+  );
```
Comment (ocrearyy) [inline src/App.js:10]: Maybe consider adding routers for the home and about components

**Review by ocrearyy — CHANGES_REQUESTED** [review]
Hi @fed1k ,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

Highlights
👍 The Application is working fine.

Required Changes ♻️
Check the comments under the review.

Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

You can also consider:

N.A.
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by ocrearyy — APPROVED** [review]
Hi @oshanedesign,

Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉

To highlight 🎯
No linter errors.✔️
Good code design and implementation. ✔️
Add all the requirements. ✔️

image


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Comment by oshanedesign** [comment]
e58

**Pull request merged**